schema/defs-linux.json

```diff
@@ -140,19 +140,7 @@
             }
         },
         "blkioWeight": {
-            "type": "integer",
-            "minimum": 10,
-            "maximum": 1000
```

Comment:
I think we [want to keep this][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc3/config-linux.md#L353